src/LitePatch/LitePatch.Services/Interfaces/ISettingsRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using LitePatch.Services.Models;
+
+namespace LitePatch.Services.Interfaces;
+
+public interface ISettingsRepository
+{
+    public ApplicationSetting ReadSettings();
+
+    public void WriteSettings(ApplicationSetting settings);
+
+    public void CheckForPath();
```

Comment:
If the implemented method is private, this can also be removed.

Reply:
Has been amended